Dockerfile

```diff
@@ -28,12 +24,15 @@ CMD ["sh", "-c", "yarn install; yarn storybook"]
 
 FROM base as ci
 
-COPY --link package.json yarn.lock ./
+COPY --link package.json yarn.lock .yarnrc.yml ./
+
 RUN set -eux; \
-	yarn
+	yarn && yarn cache clean
 
 # copy sources
 COPY --link . ./
-RUN yarn storybook:build
 
-CMD ["yarn", "serve"]
+RUN set -eux; \
+	yarn playwright install --with-deps && yarn cache clean
```

Comment:
should we install this on the `dev` container as well? then add instructions on how to run that inside docker?

Reply:
Done